tofu/geom/_core.py

```diff
@@ -3873,6 +3876,38 @@ def from_svg(
         verb=None,
         returnas=None,
     ):
+        """ Build a config from a svg file (Inkscape)
+
+        The svg shall have only:
+            - closed polygons (possibly inc. Bezier curves)
+            - an optional unique straight line (non-closed), for scaling
+
+        If Beziers curves are included, they will be discretized according to
+        resolution parameter res (absolute maximum tolerated distance between
+        points)
+
+        All closed polygons will be interpreted as:
+            - a Ves instance if it has no fill color
+            - a PFC instance if it has a fill color
+        The names are derived from Inkscape objects id
+
+        he coordinates are extracted from the svg
+        They can be rescaled either:
+            - automatically:
+                scaling computed from the unique straight line
+                and from the corresponding 2 points real-life coordinates
+                provided by the user as 2 iterables (list, arrays or tuples)
+                of len() = 2 (point1 and point2)
+                Alternatively a single point (point1) and the length of the
+                line can be provided
+            - forcefully:
+                the origin (r0, z0) and a common scaling factor (scale) are
+                provided by the user
+
+        The result Config instance must have a Name and be associated to an
+        experiment (Exp).
```

Comment:
I have forgotten the Name argument, and there is a long stack on the trace callback, maybe they shouldn't be optional

Reply:
The docstr says must, so I guess it is clear, but I agree this can be annoying.
In a future version I will try make this optional or set default values when they are forgotten, but this may take some time so I suggest we keep it as it is for now